lnbits/extensions/tpos/views_api.py

```diff
@@ -81,6 +82,26 @@ async def api_tpos_create_invoice(
     return {"payment_hash": payment_hash, "payment_request": payment_request}
 
 
+@tpos_ext.get("/api/v1/tposs/{tpos_id}/invoices")
+async def api_tpos_get_latest_invoices(tpos_id: str = None):
+    payments = [
```

Comment:
- should wrap the logic in a `try-except`, we do not want SQL errors to pop-up at the api level